src/outline.wgsl

```diff
@@ -1,5 +1,6 @@
-#import bevy_pbr::mesh_view_bindings
```

Comment:
these shader changes were due to this

https://bevyengine.org/learn/migration-guides/0.10-0.11/#improve-shader-import-model

I'm not as experienced with these shaders, so I wouldn't be surprised if there are changes here that "work" but may only work in the cases I've been testing.